mkdocs-base.yml

```
@@ -10,29 +10,34 @@ nav:
     - Context: ./introduction/context.md
     - Acknowledgements: ./introduction/acknowledgements.md
   - Language specification:
+    - About the grammar sections: ./language_specification/about_the_grammar_sections.md
     - Structure:
       - General structure: ./language_specification/structure/general_structure.md
       - File extension: ./language_specification/structure/file_extension.md
       - Case sensitivity: ./language_specification/structure/case_sensitivity.md
-      - Whitespace & Comments: ./language_specification/structure/whitespace_comments.md
-      - Identifiers: ./language_specification/structure/identifiers.md
-      - Reserved keywords: ./language_specification/structure/reserved_keywords.md
+    - Tokens:
+      - Tokens: ./language_specification/tokens/tokens.md
+      - White space: ./language_specification/tokens/white_space.md
+      - Newlines: ./language_specification/tokens/newlines.md
+      - Literals: ./language_specification/tokens/literals.md
+      - Identifiers: ./language_specification/tokens/identifiers.md
+      - Keywords: ./language_specification/tokens/keywords.md
+      - Operators and punctuators: ./language_specification/tokens/operators_and_punctuators.md
```

Tokens should be a subsection of "Structure". And the token file can be omitted because it just lists the sections that follow. "Operators and punctuators" changed to "Operators & punctuators" and the filename should also be adjusted.
```suggestion
      - Tokens:
        - White space: ./language_specification/tokens/white_space.md
        - Newlines: ./language_specification/tokens/newlines.md
        - Literals: ./language_specification/tokens/literals.md
        - Identifiers: ./language_specification/tokens/identifiers.md
        - Keywords: ./language_specification/tokens/keywords.md
        - Operators & Punctuators: ./language_specification/tokens/operators_punctuators.md
```

---

OK to the Tokens being a subsection of Structure.

Why using `& Punctuators` though?